app/controllers/commons_controller_mixin.rb

```diff
@@ -10,6 +10,30 @@ module CommonsControllerMixin
   def configure_search
     @search = model.ransack(params[:q])
 
+    @results = @search.result().order(issue_date: :desc).order(id: :desc)
```

Comment:
I think you can do this

```suggestion
    @results = @search.result().order(issue_date: :desc, id: :desc)
```